nin/backend/serve.js

```diff
@@ -48,6 +48,7 @@ const serve = async function(
     };
 
     if (filename == 'graph.json') {
+      event.name = 'graph';
```

Comment:
Actually, it's not strictly necessary.

The reasoning was that errorExcerpt.js checks whether we have an entry in the filecache for the file that currently has an error, and I was afraid that if name was undefined for graph.json the check would fail.
Turns out `undefined in object` evaluates to false anyway, so then it works either way.

Still, for some errors this value is printed in the UI I believe. When an exception occurs when graph.json is loaded and tries instantiating a node with an error in it.